### DISCUSSION TITLE
the font size has increased in fragment setting with commit "More MD3 ugly components"

### DISCUSSION THREAD

**Discussion by yashikada** [LSPosed/LSPosed#1848] [Q&A] [answered]
I see this behaviour on more than one device, android 9, 11.
After the commit 2cb973656d01f35a7e881761375e1434de510a0e the font size has increased in fragment setting, in all other fragment is the same, follow some screenshot.
I tested also latest commit. Looking at the screenshots it's not that obvious. On real screen, it is almost double.
Is it correct?

![Screenshot_20220414-173026](https://user-images.githubusercontent.com/92667160/163464014-539ba3c2-6755-4631-b7d8-a9f1fa039e64.png)
![Screenshot_20220414-173206](https://user-images.githubusercontent.com/92667160/163464018-c103c44a-1eae-4ae3-9596-da80dd061e06.png)
![Screenshot_20220414-173233](https://user-images.githubusercontent.com/92667160/163464019-b85a9bb4-5748-4937-99ed-3b65ebfb0848.png)

**Comment by yujincheng08** [comment, marked as answer]
That's correct.
